api/managementpb/checks.proto

```diff
@@ -53,4 +78,18 @@ service SecurityChecks {
       body: "*"
     };
   }
+  // ListSecurityChecks returns list of available Security Checks and their status.
+  rpc ListSecurityChecks(ListSecurityChecksRequest) returns (ListSecurityChecksResponse) {
+    option (google.api.http) = {
+      post: "/v1/management/SecurityChecks/List"
+      body: "*"
+    };
+  }
+  // ChangeSecurityChecks enables/disables Security Checks by name.
+  rpc ChangeSecurityChecks(ChangeSecurityChecksRequest) returns (ChangeSecurityChecksResponse) {
```

Comment:
```suggestion
  rpc ChangeSecurityCheck(ChangeSecurityCheckRequest) returns (ChangeSecurityCheckResponse) {
```

We already have patterns for those: `ListAgents` (plural), but `AddNodeExporter`, `ChangeNodeExporter` (singular)

Reply:
Fixed